i18n/appointments/locale_en.json

```diff
@@ -244,5 +243,8 @@
   "CHANGE_DATE_TO_LABEL": "Change ${moment(originalAppointmentDate).format('Do MMM')} to",
   "PATIENT_DOUBLE_BOOKING_CONFLICT_MESSAGE": "Current {currentAppointmentService} request conflicts with {existingAppointmentService} appointment on and",
   "NO_SERVICE_CONFLICTS_DEFAULT_TEXT": "The {label} service you had selected for the appointment(s) is not available during below listed dates",
-  "PROVIDER_ERROR_MESSAGE": "Please select only a maximum of {maxAppointmentProviders} provider(s)"
+  "PROVIDER_ERROR_MESSAGE": "Please select only a maximum of {maxAppointmentProviders} provider(s)",
+  "APPOINTMENT_BOOKING_MESSAGE": "Appointment for #patientName, #patientId on #dateTime for #service is booked at #clinicName. For any queries call us on +919876543210.",
```

Comment:
The phone number can't be hardcoded in locale file, but instead should be read from an externally configurable configuration.

Reply:
Since this message is a clinic configuration, it can be set clinic by clinic basis

Reply:
@kavitha-sundararajan  can you check this value is overridden with config specific locale key value pair

Reply:
Yes Bindu, it will get overridden with clinic-config values. As Gurpreet suggested, the help desk phoneNumber is made as global property.